backend/src/database/seeders/20220713115148-seed-users.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  up(queryInterface) {
+    return queryInterface.bulkInsert('Users', [
+      {
+        email: 'admin@rasporedar.com',
+        password: 'admin',
+        createdAt: new Date(Date.now()),
```

Comment:
`new Date(Date.now())` and `new Date()` do the same thing so latter is preffered